pkg/tool/ocm/ocm.go

```diff
@@ -17,17 +17,21 @@ type Tool struct {
 	source *github.Source
 }
 
+// NewTool builds an OCM tool object
 func NewTool() *Tool {
 	t := &Tool{
 		source: github.NewSource("openshift-online", "ocm-cli"),
 	}
 	return t
 }
 
+// Name returns the name this tool should be referenced by
 func (t *Tool) Name() string {
 	return "ocm"
 }
 
+// Install downloads this tool, verifies it's integrity via checksum, and links the newly downloaded binary
+// to the "latest" directory
 func (t *Tool) Install(rootDir, latestDir string) error {
```

Comment:
I feel like this function could be broken up to make it more easily testable. For example, a function to return the `checksumAsset` and the `ocmBinaryAsset` from the `release` could be its own function. The code in this function is already broken down into checkpoints with comments; it looks like those could fairly easily become their own testable functions.